Dockerfile

```diff
@@ -19,10 +19,13 @@ MAINTAINER devops@edx.org
 # MySQL-python for performance gains.
 
 # If you add a package here please include a comment above describing what it is used for
-RUN apt-get update && apt-get upgrade -qy && apt-get install language-pack-en locales git python3.5 python3-pip libmysqlclient-dev libssl-dev python3-dev -qy && \
+RUN apt-get update && apt-get upgrade -qy && apt-get install language-pack-en locales git python3.6 python3-pip libmysqlclient-dev libssl-dev python3.6-dev -qy && \
```

Comment:
@coryleeio Do you know if we have tested 3.6 or what is needed to get this PR merged.